### DISCUSSION TITLE
MOOSE WorkShop training Registration

### DISCUSSION THREAD

**Discussion by jiezhou1985** [idaholab/moose#26855] [Q&A General] [answered]
Hi guys, just want to know when we could have the feedback/decision on our workshop training registration. I did register for Oct. workshop at UIUC. Maybe the time is too further away? Really looking forward to participating in.

**Comment by GiudGiud** [comment]
Hello

Yes we ll typically email out one to two months ahead of time that's it.
We dont typically send auto-acceptance emails but we might start at some point

Guillaume

**Reply by jiezhou1985** [reply]
it sounds great, so, could I confidently say that we are picked? :), since earlier plan could make some cheaper flight or living expenses. Thanks!

**Reply by GiudGiud** [reply]
yes. We do not typically refuse anyone for MOOSE training (it s open source). For BISON, it's happened before

**Reply by jiezhou1985** [reply]
Thanks! Looking forward to the training!

**Reply by jiezhou1985** [reply]
Hello, since the date is approaching, not sure whether the registration confirmation is sending out? Do we have any expected date for this confirmation and guidance? Thanks!

**Reply by GiudGiud** [reply]
@permcody

**Reply by permcody** [reply, marked as answer]
@jiezhou1985 - I see that you are registered. We will send out more detailed instructions soon (within the next two weeks) on travel and an agenda. You may start looking for flights and making arrangements to attend. I have every expectation that the workshop will occur.

**Reply by jiezhou1985** [reply]
Thanks so much, looking forward to it!
